src/fu-engine.c

```diff
@@ -5944,6 +5946,16 @@ fu_engine_ensure_security_attrs(FuEngine *self)
 	/* distil into one simple string */
 	g_free(self->host_security_id);
 	self->host_security_id = fu_engine_attrs_calculate_hsi_for_chassis(self);
+
+	/* Convert Security attribute to json string */
+	data = fu_security_attrs_to_json_string(self->host_security_attrs, &error);
```

Comment:
you need to catch the NULL condition here and display a warning with the error at this time.

Reply:
Sure, I'll display a warning message here if data is NULL.